storage/kafka/kafka.go

```
@@ -94,8 +101,38 @@ func new() (storage.StorageDriver, error) {
 	return newStorage(machineName)
 }
 
+func generateTLSConfig() *tls.Config {
+	if *certFile != "" && *keyFile != "" && *caFile != "" {
+		cert, err := tls.LoadX509KeyPair(*certFile, *keyFile)
+		if err != nil {
+			glog.Fatal(err)
```

This should return the error instead.